crypto/signer.go

```diff
@@ -0,0 +1,83 @@
+package neofscrypto
+
+import (
+	"fmt"
+)
+
+// Scheme represents digital signature algorithm with fixed cryptographic hash function.
+//
+// Non-positive values are reserved and depend on context (e.g. unsupported scheme).
+type Scheme uint32
+
+//nolint:revive
+const (
+	_ Scheme = iota
```

Comment:
What's wrong with 0? I think we should stay consistent with the API where possible.

Reply:
According to package API `Scheme` enum doesn't relate to API, so I decided to make `0` value reserved (like unspecified). But I don't mind to sync them by value, if you think this is really needed.

Reply:
Well, I think this is what users would expect to see. Also zero scheme in the API plays nicely for the default value.